src/old-index.js

```diff
@@ -1,3 +1,6 @@
+//do I need this here??? is this only for the new index? TODO: what the heck
```

Comment:
Again, old files should not be added to production.

Reply:
Fixed!